Manual page build fails with cmake

Following the build instructions from the [BUILDING](/obgm/libcoap/blob/develop/BUILDING) document, cmake fails with an error message indicating that the directories `man_tmp` and `man_html` are missing in the build directory.

```shell
$ cmake ..
[...]
-- Found GnuTLS: /usr/lib/x86_64-linux-gnu/libgnutls.so (found version "3.7.1") 
-- compiling with gnutls support
-- ENABLE_DTLS:.....................ON
-- ENABLE_TCP:......................ON
-- ENABLE_DOCS:.....................ON
-- ENABLE_EXAMPLES:.................ON
-- DTLS_BACKEND:....................default
-- WITH_GNUTLS:.....................ON
-- WITH_TINYDTLS:...................OFF
-- WITH_OPENSSL:....................OFF
-- WITH_MBEDTLS:....................OFF
-- HAVE_LIBTINYDTLS:................
-- HAVE_LIBGNUTLS:..................1
-- HAVE_OPENSSL:....................
-- HAVE_MBEDTLS:....................
-- COAP_EPOLL_SUPPORT:..............1
-- CMAKE_C_COMPILER:................/usr/bin/cc
-- BUILD_SHARED_LIBS:...............OFF
-- CMAKE_BUILD_TYPE:................Debug
-- CMAKE_SYSTEM_PROCESSOR:..........x86_64
-- Found Doxygen: /usr/bin/doxygen (found version "1.9.1") found components: doxygen dot 
-- Setup up the Doxygen documention build
-- Configuring done
-- Generating done
-- Build files have been written to: /tmp/libcoap/build
$ cmake --build .
Scanning dependencies of target doc_doxygen
[  2%] Generating API documentation with Doxygen
warning: tag INPUT: input source '/tmp/libcoap/build/doc/man_tmp' does not exist
error: source '/tmp/libcoap/build/doc/man_html' is not a readable file or directory... skipping.
 (warning treated as error, aborting now)
gmake[2]: *** [CMakeFiles/doc_doxygen.dir/build.make:77: CMakeFiles/doc_doxygen] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:107: CMakeFiles/doc_doxygen.dir/all] Error 2
gmake: *** [Makefile:149: all] Error 2
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -579,6 +579,10 @@ if(ENABLE_DOCS)
     set(DOXYGEN_IN ${CMAKE_CURRENT_SOURCE_DIR}/doc/Doxyfile.in)
     set(DOXYGEN_OUT ${CMAKE_CURRENT_BINARY_DIR}/Doxyfile)
 
+    # Make necessary temporary directories
+    file(MAKE_DIRECTORY  ${CMAKE_BINARY_DIR}/doc/man_tmp)
+    file(MAKE_DIRECTORY  ${CMAKE_BINARY_DIR}/doc/man_html)
+
     # request to configure the file
     configure_file(
       ${DOXYGEN_IN}

--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -66,6 +66,7 @@ man-page-build: man-page-start
 	## Close off the insert_file
 	echo '    </tab>' >> $(top_builddir)/doc/insert_file ;\
 	echo '    <tab type="user" visible="yes" url="@ref deprecated" title="Deprecated Items" intro=""/>' >> $(top_builddir)/doc/insert_file ;\
+	echo '    <tab type="user" visible="yes" url="@ref bug" title="Bug List" intro=""/>' >> $(top_builddir)/doc/insert_file ;\
 	## Create and Update the DoxygenLayout.xml file
 	$(DOXYGEN) -l ;\
 	$(SED) -i 's/<tab type="pages" visible="yes" /<tab type="pages" visible="no" /g' $(top_builddir)/doc/DoxygenLayout.xml ;\

